ui/components/app-wrapper.js

```diff
@@ -19,13 +19,20 @@ Router.onRouteChangeError = NProgress.done;
 Router.onRouteChangeStart = NProgress.start;
 
 
-export default function wrapPage(Page, dataFn) {
+export default function wrapPage(Page, dataFn, headerFooterParams = { showSearch: true }) {
   function WrappedPage(props) {
     if (props.err) return <ErrorPage err={props.err} />;
-    return <HeaderFooter><Page {...props} /></HeaderFooter>;
+    return <HeaderFooter showSearch={ headerFooterParams.showSearch }><Page {...props} /></HeaderFooter>;
```

Comment:
Would it make sense to just pass any parameter the user requested? This way we won't have to modify this function should we add more parameters. Something like:
```js
return <HeaderFooter {...headerFooterParams}><Page {...props} /></HeaderFooter>;
```

Reply:
I had to look up how the spread syntax worked. This is brilliant.